src/mpid/common/sched/mpidu_sched.c

```diff
@@ -162,6 +162,10 @@ int MPIDU_Sched_next_tag(MPIR_Comm * comm_ptr, int *tag)
     MPIR_FUNC_ENTER;
 
     *tag = comm_ptr->next_sched_tag;
+    if (coll_group != MPIR_SUBGROUP_NONE) {
+        /* subgroup collectives use the same tag within a paraent collective */
```

Comment:
s/paraent/parent

Reply:
fixed.

Also, rebased, resolving tons of conflicts with `MPIR_ERR_COLL_CHECKANDCONT` PR. Fingers crossed.